app/src/features/reports/ChartDumpReport.tsx

```diff
@@ -77,13 +76,8 @@ function ChartDumpReport() {
             <div style={{ width: "500px", margin: "auto", textAlign: "left" }}>
               <h1>Time Series & Two Variable Bar Chart</h1>
             </div>
-            <CovidReport
-              variable={"covid_deaths_pct_of_geo"}
-              geography={STATE_FIPS_MAP[1]}
-            />
-
+            <CovidReport variable="covid_cases" stateFips="00" />
```

Comment:
Use a constant? Especially since this isn't a real FIPS code

Reply:
good call